src/testing/dual_weighted_residual_mesh_adaptation.cpp

```diff
@@ -42,6 +42,7 @@ int DualWeightedResidualMeshAdaptation<dim, nstate> :: run_test () const
     const Parameters::AllParameters param = *(TestsBase::all_parameters);
     using ManParam = Parameters::ManufacturedConvergenceStudyParam;
     ManParam manu_grid_conv_param = param.manufactured_convergence_study_param;
+    bool check_for_p_refined_cell = (param.mesh_adaptation_param.p_refine_fraction > 0) ? true : false;
```

Comment:
```suggestion
    bool check_for_p_refined_cell = (param.mesh_adaptation_param.p_refine_fraction > 0);
```

Reply:
The above should be equivalent to the original line using the conditional operator

Reply:
Yes, I can confirm that it is

Reply:
Wow, thanks for pointing that out. It's fixed in the recent commit.